zdns/main.go

```diff
@@ -139,6 +142,37 @@ func main() {
 		gc.NameServers = strings.Split(*servers_string, ",")
 		gc.NameServersSpecified = true
 	}
+	if *localaddr_string != "" {
+		for _, la := range strings.Split(*servers_string, ",") {
+			ip := net.ParseIP(la)
+			if ip != nil {
+				gc.LocalAddrs = append(gc.LocalAddrs, ip)
+				log.Info("using local address: ", localaddr_string)
```

Comment:
This will print out multiple times, which doesn't seem correct, given that it will be printing out the entire CSV of the input.

Reply:
Good catch. Moved the info-log behind the loop.